### PR TITLE
Modularize Components and Define them all in one file

### PR DESCRIPTION
**Pull request by chintannp**
I have modularized the components. Do you think we should have a separate component defined for the AddTodoButton?

```jsx
    <View style={styles.container}>
      <Header />
      <AddModal modalVisible={modalVisible} setModalVisible={setModalVisible} />
      <TodoList todos={todos}/>
      <Pressable
        style={[styles.buttonContainer, styles.floatingButton]}
        onPress={() => {
          setModalVisible(true);
        }}>
        <Text style={styles.buttonText}>+ Add Todo</Text>
      </Pressable>
    </View>

```

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
 import React from 'react';
-import { StyleSheet, SafeAreaView,StatusBar } from 'react-native';
+import { StyleSheet, SafeAreaView, StatusBar } from 'react-native';
 import Amplify from 'aws-amplify'
 import awsconfig from './src/aws-exports'
 import Home from './src/Home';
-import Header from './src/Header';
```
Comment (calebpollman) [inline App.js:6]: Did you delete this file as well? Not seeing it in the diff

File: src/Home.js [hunk 2]
```diff
@@ -47,39 +24,11 @@ export default function Home() {
...
-          setComplete(!item.isComplete, item)
-        }}
-      >
-        <Text style={[styles.checkboxText, item.isComplete && styles.selectedCheckboxText]}>{(item.isComplete) ? "✓" : ""}</Text>
-      </Pressable>
-    </Pressable >
-  );
-
-  const addTodoModal = (<Modal
+  return (<Modal
```
Comment (calebpollman) [inline src/Home.js]: This formatting is still off, there should be a new line between `return (` and `<Modal`. Prettier should be automatically fixing these issues?

File: src/Home.js [hunk 3]
```diff
@@ -105,16 +54,84 @@ export default function Home() {
...
+    />)
+};
+
+
+export default function Home() {
+
+  const [todos, setTodos] = useState([]);
+  const [modalVisible, setModalVisible] = useState(false);
+
+  useEffect(() => {
```
Comment (calebpollman) [inline src/Home.js]: Looks like this `useEffect` could just be in `TodoList`?

**Review by calebpollman — CHANGES_REQUESTED** [review]
This looks good for the most part, but seeing quite a few formatting issues, all of which should be picked up by prettier. You may want to double check that your installation is correct

File: src/Home.js (hunk 3 above)
Comment (chintannp) [reply src/Home.js]: Good point. Makes the home component much smaller.

File: src/Home.js (hunk 2 above)
Comment (chintannp) [reply src/Home.js]: Prettier is working fine. If I define the home file with jsx extension this is formatting as the way you defined above. However, with .js extension it formats like above. I have fixed the formatting as of now.

File: App.js (hunk 1 above)
Comment (chintannp) [reply App.js:6]: Deleted it now.

File: src/Home.js [hunk 4]
```diff
@@ -1,22 +1,85 @@
...
+  }
+
+  function closeModal() {
+    setModalVisible(false);
+  }
+
+  return (
+    <Modal
+      animationType="fade"
+      transparent={true}
```
Comment (calebpollman) [inline src/Home.js]: Nit: [use JSX shorthand](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true) for declaring `true` prop

File: src/Home.js [hunk 5]
```diff
@@ -1,22 +1,85 @@
...
-  const [modalVisible, setModalVisible] = useState(false);
 
   useEffect(() => {
     const onQuery = async () => {
       const todos_list = await DataStore.query(Todo);
       setTodos(todos_list);
     };
 
-    //load todos on first render 
+    //load todos on first render
```
Comment (calebpollman) [inline src/Home.js]: There should always be a space between the content of a comment and the `//`

File: src/Home.js [hunk 6]
```diff
@@ -1,22 +1,85 @@
...
+  Text,
+  View,
+  FlatList,
+  Pressable,
+  Modal,
+  TextInput,
+  TouchableOpacity,
+} from 'react-native';
+import {DataStore} from 'aws-amplify';
+import {Todo} from './models';
```
Comment (calebpollman) [inline src/Home.js]: Why are we no longer leaving white space between the brackets?

File: src/Home.js (hunk 4 above)
Comment (chintannp) [reply src/Home.js]: Thanks. This is really helpful

File: src/Home.js (hunk 5 above)
Comment (chintannp) [reply src/Home.js]: Got it.

File: src/Home.js (hunk 6 above)
Comment (chintannp) [reply src/Home.js]: Prettier removed it for jsx file extension. I have fixed it.

**Review by calebpollman — APPROVED** [review]

**Pull request merged**